test/e2e/provisioning/helper.go

```diff
@@ -113,7 +113,7 @@ func runScenarios(t *testing.T, excludeSelector *scenarioSelector, testParams st
 			machineName := fmt.Sprintf("machine-%s", nameSufix)
 			nodeName := fmt.Sprintf("node-%s", nameSufix)
 			params := testParams
-			params = fmt.Sprintf("%s,<< MACHINE_NAME >>=%s,<< NODE_NAME >>=%s", params, machineName, nodeName)
+			params = fmt.Sprintf("%s,<< MACHINE_NAME >>=%s,<< NODE_NAME >>=%s-%s", params, machineName, nodeName, os.Getenv("CIRCLE_BUILD_NUM"))
```

Comment:
Not particular pretty as you now have circle-ci specifics in here, but fine for now.
But something we should clean up at some point